src/components/toggle.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+import HandleView from './view.jsx';
+
+class ToggleContainerViews extends React.Component {
+  constructor(props) {
+    super();
+
+    const view = props.defaultView;
+
+    this.state = {
+      view,
+    };
+  }
+
+  handleClick(view) {
+    this.setState({
+      view,
```

Comment:
We are thinking of handling state in a single object contained in the app component. If we go with this approach then the function to change this will be passed down as a prop to each component. We will discuss this on Monday though.